tests/test_tavern.py

```diff
@@ -1195,6 +1195,7 @@ def test_kaelthas_sunstrider(self):
         tavern = Tavern()
         player_1 = tavern.add_player_with_hero("Dante_Kong", KaelthasSunstrider())
         player_2 = tavern.add_player_with_hero("lucy")
+        tavern.randomizer = RepeatedCardForcer([DragonspawnLieutenant, ScavengingHyena, Scallywag, VulgarHomunculus])
```

Comment:
Even if this is intentional, it's surprising here.  Note the drawn hands will be:

Note it will be 
Turn 1:
Player 1:  [1,2,3]
Player 2: [4,1,2]
Turn 2
Player 1: [3,4,1]
Player 2: [2,3,4]
Turn 3:
Player 1:  [1,2,3]
Player 2: [4,1,2]
Turn 4:
Player 1: [3,4,1]
Player 2: [2,3,4]


Note that turns 1 and 3 draw the same cards, as do turns 2 and 4.  This works out since it takes three cards to triple, but someone reading this code would be confused, without a comment making clear that you are expecting this behavior, rather than drawing all DragonspawnLieutenant on the first turn, ScavengingHyena on the second turn, etc.

Reply:
Makes sense. I think I'll make a new randomizer each turn, which should make the intended behavior more clear.